### PR TITLE
feat(core): Add chainId to genesis header

### PR DESCRIPTION
**Pull request by stbrody**

### CODE REVIEW

File: packages/http-client/src/ceramic-http-client.ts [hunk 1]
```diff
@@ -131,6 +132,17 @@ class CeramicClient implements CeramicApi {
     }
   }
 
+  async getChainId(): Promise<string> {
```
Comment (oed) [inline packages/http-client/src/ceramic-http-client.ts:135]: I think we should change this to something like this: 
```suggestion
  async getSupportedChains(): Promise<Array<string>>
```
This way we don't need to introduce a breaking change when nodes can support multiple anchor services.

File: packages/common/src/anchor-service.ts [hunk 2]
```diff
@@ -29,4 +29,9 @@ export abstract class AnchorService extends EventEmitter {
      */
     abstract validateChainInclusion(anchorProof: AnchorProof): Promise<void>;
 
+    /**
+     * @returns a string representing the CAIP-2 chain ID of the configured blockchain.
+     */
+    abstract getChainId(): Promise<string>;
```
Comment (oed) [inline packages/common/src/anchor-service.ts:35]: I guess we need to change it here as well :)

**Review by oed — CHANGES_REQUESTED** [review]
Looks good, one change I'd like in here.

File: packages/http-client/src/ceramic-http-client.ts (hunk 1 above)
Comment (stbrody) [reply packages/http-client/src/ceramic-http-client.ts]: Good idea!  New version up that does this @oed.  Also updated the CAS side to also return an array in https://github.com/ceramicnetwork/ceramic-anchor-service/pull/151

**Review by oed — APPROVED** [review]

**Review by simonovic86 — APPROVED** [review]
lgtm!

**Pull request merged**